### DISCUSSION TITLE
Using API

### DISCUSSION THREAD

**Discussion by fire0086** [amirinsight/py-bingx#7] [Q&A]
Hello Sir,
I don't know how to use the API yet, can you give me a guide to launching the API?
Thank you

**Comment by amirinsight** [comment]
Hi,
Did you try following the instructions?

**Comment by tolaku** [comment]
>If this used to work before, it might be due to some changes on bingx api.

Вы писали в другой ветке - что связано с некоторыми изменениями в bingx api. Как в моем случае нужно поступить, подскажите пожалуйста?

**Reply by amirinsight** [reply]
1. When was the last time when it was working correctly?

2. Have you tried using server timestamps? (timestamp="server")

**Comment by tolaku** [comment]
1. 25 апреля
2. А как это сделать, подскажите пожалуйста? Благодарю.

**Reply by amirinsight** [reply]
1. There has been no major changes to the API since then.

2. Please read py-bingx readme carefully. I have explained this in the "Usage" part.

**Reply by tolaku** [reply]
Вы мне очень помогли, прочитал подробно README.rst, и вы там рекомендовали timestamp="server". Теперь получают ответ положительный (OK). Очень Вам благодарен за терпение и помощь. 

p.s. А из-за чего так произошло, если ранее при timestamp="local" все работало, почему сейчас нужно менять на timestamp="server", просто хочу понять? Благодарю.

**Reply by amirinsight** [reply]
Congrats on making it work.

This is probably due to your laptop's time being inaccurate.
